lib/widgets-api.php

```diff
@@ -37,6 +37,18 @@ function gutenberg_assign_widget_to_sidebar( $widget_id, $sidebar_id ) {
 	wp_set_sidebars_widgets( $sidebars );
 }
 
+function gutenberg_find_widgets_sidebar( $widget_id ) {
```

Comment:
This needs a doc comment. Also, I need to update the controller to use it so that there's not two of the same function.